graderx/graders/app_config.py

```diff
@@ -0,0 +1,19 @@
+from pathlib import Path
+
+
+COURSES_DATA_PATH = Path(__file__).parent.joinpath("courses_config.json")
+LAB_GUIDE_FILENAME = 'lab_guide.md'
+MOSS_PATH = Path(__file__).parent.joinpath('moss/submissions')
```

Comment:
we shouldn't manually join paths using `/` as this is not cross platform. (windows uses `\`), since python itself is cross platform, we should use the built in python APIs `os.path.join` that will handle those cases for us